src/modules/Layout/Container/Container.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
```

Comment:
@nik-kor just removed `Container` and put alignment stuff in **css**.

Thank you for an advice!